deliverables/README.md

```diff
@@ -11,17 +11,17 @@ Sign Up for Cloud9 and create a basic Rails project.
 ###Intro to Ruby
 ####Required
 
-What is an "interpreted" language?<br>
-What is IRB?<br>
-What are Objects?<br>
-What are Methods?<br>
-What are Classes?<br>
-What are Blocks?<br>
-What is an Array?<br>
-What is an Iterator?<br>
-What are hashes?<br>
-What is a library?<br>
-What is a gem?
+What is an "interpreted" language?<br> It is a language that doesn't need a compiler to read and write.
+What is IRB?<br> Interactive Ruby Shell, it allows you to write and test Ruby from a terminal.
+What are Objects?<br> A data structure that makes up nearly everything: variables, numbers, arrays, lambdas, procs, other than things like blocks.
+What are Methods?<br> What objects call upon in other objects to communicate with each other.
+What are Classes?<br> Classes are structures that hold behavior and state.
+What are Blocks?<br>A block is a contained bit of code that can't be stored in a variable, but can be stored in a proc and lambda, can only be called upon once per argument, and isn't an object.<br>
+What is an Array?<br> An array is used to store a list of values in a single variable, indices start at 0, can store numbers, strings, booleans, other arrays.
```

Comment:
"can only be called upon once per argument" Not sure what you mean. Block can be called an infinite number of times in fact a block can call itself. This is called recursion.